packages/cli/src/Server.ts

```diff
@@ -320,9 +320,7 @@ export class Server extends AbstractServer {
 				limit: 0,
 			},
 			banners: {
-				v1: {
-					dismissed: false,
-				},
+				permanentlyDismissed: [],
```

Comment:
do we plan to add temporarily dismissed banners? if not, I'd suggest we rename this to just `dismissed` to prevent any confusion.

Reply:
No, on back-end there are only these.